README.rst

```diff
@@ -161,7 +161,9 @@ To report security issues please send an email to
 .. |Build Status| image:: https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=main
    :target: https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=main
 .. |MIT license| image:: https://img.shields.io/badge/License-MIT-blue.svg
+   :target: https://github.com/fairlearn/fairlearn/blob/main/LICENSE
 .. |PyPI| image:: https://img.shields.io/pypi/v/fairlearn?color=blue
+   :target: https://badge.fury.io/py/fairlearn
```

Comment:
Again, i'm not sure why not link to https://pypi.org/project/fairlearn/

Reply:
Same as above, replacing this shortly 🙂 Thanks!